Abracadabra/abracadabra_web_app/pages/profile.js

```diff
@@ -1,14 +1,14 @@
 import { Nav } from "react-bootstrap";
+import BlankNavBar from "src/components/BlankNavBar.js";
 import EditUser from "../src/components/EditUser.js";
-import Navbar from "../src/components/Navbar.js";
 
-function EditUserPage() {
+function profile() {
   return (
     <>
-      <Navbar />
+      <BlankNavBar />
       <EditUser />
     </>
   );
 }
 
-export default EditUserPage;
+export default profile;
```

Comment:
This is going to change, when we add the public profiles, but it is fine for now.